docs/Project.toml

```diff
@@ -4,3 +4,4 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 
 [compat]
 Documenter = "~0.22"
+Gadfly = "1"
```

Comment:
```suggestion
Gadfly = "1"

```

Reply:
should I go ahead commit this suggestion?

Reply:
yes thank you, I don't think I can commit directly to your PR